### DISCUSSION TITLE
PK-sim for antibodies?

### DISCUSSION THREAD

**Discussion by rituran** [Open-Systems-Pharmacology/Forum#1676] [Questions & Problems]
Hello! 
I want to use PK-Sim to get the PK/PD and dosing concentrations for antibodies being developed. 
What all input parameters are needed for using PK-sim for antibodies specifically? This is my first time using PK-Sim, so if anybody has successfully used it for the same purpose, please let me know. Thanks!

**Comment by StephanSchaller** [comment]
Hi Ritu @rituran ,

certainly, we have extensive expertise in utilizing PK-Sim for biologics PK/PD assessments.
Most crucial will be your molecular weight, FcRn binding affinity in endosomes, and possibly target binding kinetics (Kd, koff) and kinetics of the target itself (expression levels, turnover rates), should it be relevant for target-mediated clearance.

Best,
Stephan

**Reply by rituran** [reply]
@StephanSchaller thanks for your response. So just the clarify, these all parameters to input would be using in vitro/ lab based assays right? And also how close are the results from PK-Sim to the results obtained from in vivo/ human trials?

**Reply by StephanSchaller** [reply]
Yes, in-vitro, ideally. Or can be assumed from likeliness to wild-type IGG.

For accuracy, see the publication, here:
https://link.springer.com/article/10.1007/s10928-017-9559-4

**Reply by Christoph27** [reply]
Just a note to Kd(FcRn) in endosomal space since this is a tricky one: The measured Kd(FcRn) values are very assay dependent (i.e. dependent on assay details like if Ab or FcRn is immobilized on the chip, etc.). A certain set of Kd(FcRn) values were used during model development (especially for fitting endosomal FcRn concentrations), cf. https://doi.org/10.1007/s10928-017-9559-4 where e.g. a Kd(FcRn) = 0.63 µM or 0.85 µM is used for human/humanized Ab in human. 
Usually my approch is like this: For a human/humanized Ab in human, I start by setting Kd(FcRn) to 0.85 µM (from my experience I prefer the 0.85 µM over the 0.63 µM) and fit other CL pathways e.g. target mediated CL. 
Or to put it differently: if you set Kd(FcRn) = 0.85 µM, your antibody in PK-Sim behaves like a typical human/humanized Ab in human without any further CL pathways.

**Comment by Christoph27** [comment]
Hi Ritu, 
just as an addendum to Stephan's answer above: For target mediated clearance, there is usually also a degradation rate constant for the antibody-target-complex. The processes of target turnover and degradation of the antibody-target-complex are added in MoBi. 
For the "base" model in PK-Sim there is also the solute radius, which is calculated from the molecular weight by default. You can also enter directly an experimental value for this. 5.34 nm were used for antibodies during the development of the large molecules model in PK-Sim.

**Reply by rituran** [reply]
@Christoph27 thanks for the response. I am actually trying to determine what all assays I would need in the lab to get the input parameters for PK-sim, and for the degradation rate constant for antibody-target-complex- would you happen to know what assay would be done to get that ?

**Reply by Christoph27** [reply]
There are assays using a fluorescence dye for Ab-labelling and measuring internalization into cells. But often a combined internalization/degradation rate constant is just fitted to PK data.

**Reply by rituran** [reply]
@Christoph27 that's good to know. Since I have never used PK-Sim, could you please explain the last part of your answer? For fitting the combined internalization/degradation rate constant to PK data, how will I get its value from an assay?

**Reply by Christoph27** [reply]
The last part of my answer meant: The parameter is not obtained via an in-vitro assay but estimated from the measured PK data by parameter identification, i.e. varying the parameter in order to get optimal matching of simulated concentration-time profiles to observed concentration-time profiles (e.g. using the parameter identification tools in PK-Sim or MoBi).
